specs/sifive.com/PRCI/RESET/0.1.0/RESET_rtl.json5

```diff
@@ -0,0 +1,42 @@
+{
+  abstractionDefinition: {
+    vendor: 'sifive.com',
+    library: 'PRCI',
+    name: 'RESET_rtl',
+    version: '0.1.0',
+    busType: {
+      vendor: 'sifive.com',
+      library: 'PRCI',
+      name: 'RESET',
+      version: '0.1.0',
+    },
+    ports: {
+      RESET: {
+        description: 'reset signals',
+        wire: {
+          isReset: true,
+          onMaster: {direction: 'out', presence: 'required'},
+          onSlave:  {direction: 'in', presence: 'required'}
+        }
+      }
+    },
+    props: {
+      ResetPolarity: {
+          type: "string",
+	  enum: ["ActiveHigh", "ActiveLow"],
```

Comment:
We should have a safe default. Something `default: 'ActiveLow'`

Reply:
sure - how does one specify that?